skipper.go

```diff
@@ -1606,6 +1606,7 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 			log.Infof("Use endpoints %s/%s to fetch updated redis shards", o.KubernetesRedisServiceNamespace, o.KubernetesRedisServiceName)
 
 			kdc := findKubernetesDataclient(dataClients)
+
 			if kdc != nil {
 				redisOptions.AddrUpdater = getRedisUpdaterFunc(o.KubernetesRedisServiceNamespace, o.KubernetesRedisServiceName, kdc)
```

Comment:
there should be error handling or returning here and with `updateEndpointsFromURL`